app/buyers/views/buyers.py

```diff
@@ -310,6 +305,45 @@ def read(self):
     ), 200
 
 
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/publish', methods=['GET', 'POST'])
+def publish_brief(framework_slug, lot_slug, brief_id):
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client,
+                                           status='live', must_allow_brief=True)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id):
+        abort(404)
+    brief_users = brief['users'][0]
+    brief_name = brief_users['name']
+    email_address = brief_users['emailAddress']
+
+    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
+        {'lot': lot['slug']}
+    )
+    sections = content.summary(brief)
+    flattened_brief = get_flattened_brief(sections)
+
+    unanswered_required, unanswered_optional = count_unanswered_questions(sections)
+    if request.method == 'POST':
+        if (unanswered_required > 0):
+            abort(404)
+        data_api_client.update_brief_status(brief_id, 'live', brief_name)
```

Comment:
I think the user should be the currently logged in user. For now this will be the same as the brief user, but in future there may be multiple users per brief - the auditing should capture who was logged in when the button was pressed.